pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
+
+- Output is formatted for a display columns wide:
+
+`printf "header1 header2\nbar foo\n" | column -c {{10}}`
+
+- Specify a set of characters to be used to delimit columns for the -t option, default is whitespace:
+
+`printf "header1,header2\nbar,foo\n" | column -s,`
+
+- Determine the number of columns the input contains and create a table:
```

Comment:
It seems this is copied from the man page verbatim. My suggestion - "Determine the  no. of columns and auto-align the output in a tabular format. "